backend/btrixcloud/pages.py

```diff
@@ -120,6 +120,7 @@ async def add_crawl_wacz_filename_to_pages(self, crawl_id: str):
                         continue
 
                     page_id = page_dict.get("id")
+                    depth = page_dict.get("depth")
```

Comment:
This won't actually set the depth currently - we'll need to switch from batch updates to per-page updates to add information that varies from page to page (batch update was possible because the filename is consistent across all pages within that WACZ file)